cmdline/bin/dcat.dart

```diff
@@ -36,34 +36,35 @@ void main(List<String> arguments) {
   dcat(paths, argResults[LINE_NUMBER]);
 }
 
-Future dcat(List<String> paths, bool showLineNumbers) {
+Future dcat(List<String> paths, bool showLineNumbers) async {
```

Comment:
it's sufficient to do:

   dcat(List<String> paths, bool showLineNumbers) async {

Without specifying the Future return type.

Not sure what the right convention is.